tests/core/test_market.py

```
@@ -3,26 +3,24 @@
-    first_match_bet_market,
-    inj_token,
-    inj_usdt_spot_market,
-    usdt_perp_token,
-    usdt_token,
-)
+from tests.model_fixtures.markets_fixtures import btc_usdt_perp_market  # noqa: F401
+from tests.model_fixtures.markets_fixtures import first_match_bet_market  # noqa: F401
+from tests.model_fixtures.markets_fixtures import inj_token  # noqa: F401
+from tests.model_fixtures.markets_fixtures import inj_usdt_spot_market  # noqa: F401
+from tests.model_fixtures.markets_fixtures import usdt_perp_token  # noqa: F401
+from tests.model_fixtures.markets_fixtures import usdt_token  # noqa: F401; noqa: F401
```

Ruff (0.11.9)

`tests.model_fixtures.markets_fixtures.usdt_token` imported but unused

Remove unused import: `tests.model_fixtures.markets_fixtures.usdt_token`

(F401)